rustworkx/rustworkx.pyi

```diff
@@ -1048,6 +1048,12 @@ def graph_union(
     merge_edges: bool = ...,
 ) -> PyGraph[_S, _T]: ...
 
+# Dominance
+
+def digraph_immediate_dominators(
+    graph: PyDiGraph[_S, _T], start_node: int, /
+) -> Mapping[int, int]: ...
```

Comment:
I think putting the return type here as `dict[int,int]` is more appropriate. The method is not generic PyO3 always cast the HashMap to a dictionary

Reply:
Done. Also changed the return type in the docstring.